cmake/Modules/FindSYCL.cmake

```diff
@@ -215,10 +215,17 @@ macro(SYCL_WRAP_SRCS sycl_target generated_files)
   set(SYCL_flags "")
   set(generated_extension ${CMAKE_${SYCL_C_OR_CXX}_OUTPUT_EXTENSION})
 
+  set(library_name)
+  if(${sycl_target} STREQUAL "xpu_ops_kernels")
+    set(library_name torch_xpu_ops_sycl_kernels)
+  else()
+    set(library_name ${sycl_target})
+  endif()
+  
   set(SYCL_include_dirs "${SYCL_INCLUDE_DIR}")
-  list(APPEND SYCL_include_dirs "$<TARGET_PROPERTY:${sycl_target},INCLUDE_DIRECTORIES>")
+  list(APPEND SYCL_include_dirs "$<TARGET_PROPERTY:${library_name},INCLUDE_DIRECTORIES>")
 
-  set(SYCL_compile_definitions "$<TARGET_PROPERTY:${sycl_target},COMPILE_DEFINITIONS>")
+  set(SYCL_compile_definitions "$<TARGET_PROPERTY:${library_name},COMPILE_DEFINITIONS>")
```

Comment:
The logic of customizing libname should be out of the file. `FindSYCL.cmake` is a file to  provide general helpers.